### ISSUE TITLE
Check that twitter handle exists before adding

### ISSUE THREAD

**Issue by rosofo** [walidvb/twitterdelights#6]
The user should be notified that the handle they've entered doesn't exist.

**Comment by walidvb** [comment]
Good point. I'm slightly concerned about the [twitter limitations](https://developer.twitter.com/en/docs/twitter-api/rate-limits#table).
One way to go around this would be to have users login via twitter, which would increase that limit.

Still, probably not an issue as long as we don't have too many visitors
